src/sphinx/code/SqlToSlick.scala

```diff
@@ -43,7 +43,7 @@ object SqlToSlick extends App {
   val db = Database.forConfig("h2mem1")
   try {
 
-    Await.result(db.run(Action.seq(
+    Await.result(db.run(DBIO.seq(
```

Comment:
I know fold and sequence. I am not a big fan of having two methods in the api where you can't distinguish what they do by name, only by memory. How about having a method `discard` instead which you can chain after sequence?